addons/website/static/src/client_actions/website_preview/website_preview.js

```diff
@@ -348,12 +348,14 @@ export class WebsitePreview extends Component {
             const { href, target, classList } = linkEl;
             if (classList.contains('o_add_language')) {
                 ev.preventDefault();
+                // TODO: in master adapt the href in template to only be the
```

Comment:
@qsm-odoo Tiny one
The real fix (to be done in master) is to alter the href in website template [here](https://github.com/odoo/odoo/blame/83e04466ff23027381166d584589966103f89cab/addons/website/views/website_templates.xml#L1931) so the js just forward that href to the wizard without the (now) useless URL part of the backend (and also remove the safety net fallback `||`
```diff
<a t-attf-class="o_add_language d-none #{dropdown and 'd-sm-block dropdown-item' or 'd-sm-inline-block list-inline-item'}"
       groups="website.group_website_restricted_editor"
-       t-attf-href="/web#action=base.action_view_base_language_install&amp;website_id=#{website.id if website else ''}&amp;url_return=#{quote_plus(url_for('', '[lang]') + '?' + keep_query())}">
+       t-att-href="quote_plus(url_for('', '[lang]') + '?' + keep_query())">
```